tests/Agent/IntegrationTests/UnboundedIntegrationTests/MySql/MySqlConnectorTests.cs

```diff
@@ -14,47 +15,55 @@
 
 namespace NewRelic.Agent.UnboundedIntegrationTests.MySql
 {
-    public abstract class MySqlConnectorTestBase : NewRelicIntegrationTest<RemoteServiceFixtures.MySqlConnectorBasicMvcFixture>
+    public abstract class MySqlConnectorTestBase<TFixture> : NewRelicIntegrationTest<TFixture> where TFixture : ConsoleDynamicMethodFixture
     {
-        private readonly RemoteServiceFixtures.MySqlConnectorBasicMvcFixture _fixture;
+        private readonly ConsoleDynamicMethodFixture _fixture;
         private readonly string _transactionName;
 
-        protected MySqlConnectorTestBase(RemoteServiceFixtures.MySqlConnectorBasicMvcFixture fixture, ITestOutputHelper output, Action<RemoteServiceFixtures.MySqlConnectorBasicMvcFixture> exerciseApplication, string transactionName) : base(fixture)
+        protected MySqlConnectorTestBase(TFixture fixture, ITestOutputHelper output, string transactionName) : base(fixture)
```

Comment:
I find the name `transactionName` for that parameter to be a little confusing since it's not just a transaction name, it's also the database command type being run in the test.  I'd suggest something like "dbOperationType" or even just "testAction".